src/index.ts

```diff
@@ -43,12 +43,9 @@ function processNoInput<T>(arg: T): T {
  * Prints the current version
  */
 function printVersion(): void {
-  if (
-    process.argv.indexOf('-v') === -1 &&
-    process.argv.indexOf('--version') === -1
-  ) {
+  if (process.argv.includes('-v') && process.argv.includes('--version')) {
```

Comment:
You probably meant "||".

What I don't really like about this is that the function is called `printVersion`, but now we're only printing it in the debug mode. I think I understand the reasoning behind this (so that the output of `targets` is machine-readable), but then at least let's print in debug unconditionally?

Reply:
>You probably meant "||".

Yup, I did. :)

>I think I understand the reasoning behind this (so that the output of targets is machine-readable), but then at least let's print in debug unconditionally?

Yup but I am not sure what you mean by "unconditionally". That should already be the case?

Reply:
Hmm, wait a sec, what's the expected behavior for you here?
Until now, this function always printed the version unless the "--version" (or the short version) was provided, because in those cases we don't want to print the version twice.
And I'd preserve that logic (just changing the level to debug).

Reply:
>And I'd preserve that logic (just changing the level to debug).

Isn't that what I'm doing here? Unless `-v` or `--version` is passed, always log but at `debug` level.